src/EdpGithub/Api/AbstractApi.php

```diff
@@ -12,6 +12,11 @@
      */
     protected $client;
 
+    /**
+     * @var ServiceManager
+     */
+    private $serviceManager;
```

Comment:
Ohh yes.. I know the theory :P

Gianluca Arbezzano
Il 03/mar/2015 17:58 "Andreas Möller" <notifications@github.com> ha scritto:

> In src/EdpGithub/Api/AbstractApi.php
> <https://github.com/EvanDotPro/EdpGithub/pull/39#discussion_r25704385>:
>
> > @@ -12,6 +12,11 @@
> >       */
> >      protected $client;
> >
> > +    /**
> > +     * @var ServiceManager
> > +     */
> > +    private $serviceManager;
>
> The places where this property is accessed are the getters and setters -
> other than that, it's never accessed directly!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EvanDotPro/EdpGithub/pull/39/files#r25704385>.
>